frontend/src/components/pages/topics/Tab.Messages.tsx

```diff
@@ -404,6 +406,14 @@ export class TopicMessageView extends Component<{ topic: TopicDetail }> {
             },
             {
                 width: 1, title: ' ', key: 'action', className: 'msgTableActionColumn',
+                filters: [],
```

Comment:
Not sure I like the way the settings are accessed right now.
Should probably be its own button or something...

But we can easily change fix that later, so its alright for now 😄